exercises/hello-world/hello-world.ts

```diff
@@ -0,0 +1,7 @@
+class HelloWorld {
+    static hello() {
```

Comment:
Params?

Reply:
I figured we'd leave those to the user. What do you think?

Reply:
I think it might be confusing not to have them. What about a c style comment inside the parens saying something like: Parameters go here.

Reply:
Sounds good. I'll have a commit up in a sec